src/App/Header/Connect.svelte

```diff
@@ -0,0 +1,236 @@
+<script lang="ts">
+  import debounce from "lodash/debounce";
+
+  import { closeFocused } from "@app/components/Floating.svelte";
+  import { sessionStore, disconnect } from "@app/lib/session";
+  import { toClipboard } from "@app/lib/utils";
+
+  import Avatar from "@app/components/Comment/Avatar.svelte";
+  import Button from "@app/components/Button.svelte";
+  import Floating from "@app/components/Floating.svelte";
+  import Icon from "@app/components/Icon.svelte";
+
+  function formatId(id: string) {
+    return id.substring(0, 4) + " – " + id.substring(id.length - 4, id.length);
+  }
+
+  let icon: "clipboard-small" | "checkmark-small" = "clipboard-small";
+
+  const restoreIcon = debounce(() => {
+    icon = "clipboard-small";
+  }, 800);
+
+  function copyToClipboard(clipboard: string) {
+    toClipboard(clipboard);
+    icon = "checkmark-small";
+    restoreIcon();
+  }
+
+  $: command = import.meta.env.PROD
+    ? "rad web"
+    : `rad web --frontend ${new URL(import.meta.url).origin}`;
+</script>
+
+<style>
+  .dropdown {
+    align-items: center;
+    background: var(--color-background-1);
+    border-radius: var(--border-radius);
+    box-shadow: var(--elevation-low);
+    color: var(--color-foreground-6);
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    right: 5rem;
+    top: 5rem;
+    width: 16.5rem;
+  }
+  .info {
+    align-items: flex-start;
+    padding: 1rem;
+    width: 20.5rem;
+  }
+  .cmd {
+    background-color: var(--color-foreground-2);
```

Comment:
Maybe we should increase the contrast here?
```suggestion
    background-color: var(--color-foreground-3);
```

In the light theme it's nice, but in the dark one, there is almost no difference between the tooltip and the background of the cmd

**current**
<img width="358" alt="Bildschirm­foto 2023-02-09 um 14 13 53" src="https://user-images.githubusercontent.com/7912302/217888699-9f5ccf29-1fa7-49b9-b45d-c41c05d1e43a.png">
**proposed**
<img width="345" alt="Bildschirm­foto 2023-02-09 um 14 16 54" src="https://user-images.githubusercontent.com/7912302/217888810-9e5ac711-3654-4f5f-a723-8b738420ba28.png">

Reply:
f6bc61813eb7e280edf5c4ae3fccfe610bffd975